ecs-microservice/main.tf

```diff
@@ -132,18 +132,26 @@ resource "aws_api_gateway_rest_api" "api_gateway_microservice_rest_api" {
 
 resource "aws_api_gateway_deployment" "api_gateway_microservice_rest_api_deployment_v1" {
   rest_api_id = aws_api_gateway_rest_api.api_gateway_microservice_rest_api.id
-  stage_name  = "v1"
-  variables = {
-    hash = sha256(var.schema)
-  }
   lifecycle {
     create_before_destroy = true
   }
 }
 
+resource "aws_api_gateway_stage" "api_gateway_microservice_stage_v1" {
+  rest_api_id = aws_api_gateway_rest_api.api_gateway_microservice_rest_api.id
+  stage_name  = "v1_new"
```

Comment:
Har ikke klart å finne noen måte å bytte over til en stage ressurs og samtidig beholde samme navn som tidligere, så virker som man må ha et nytt navn her. Om noen har forslag til et bra navn så fyr løs. 

En annen diskusjon er hvorvidt vi i det hele tatt trenger å referere til noe versjon i stage navnet. I eksempler fra aws dokumentasjon så brukes stages til forskjellige miljøer og ikke til versjoner og man har stage prod, test osv.

Reply:
jeg tror det benyttes for begge deler litt avhengig av hva du trenger.
feks hvis du har et litet system med kun en api-gw for alle miljø.